### ISSUE TITLE
Create device information in HA

### ISSUE THREAD

**Issue by jontofront** [pblxptr/ecoNET-300-Home-Assistant-Integration#9]
http://x.x.x.x/econet/sysParams

**Controller**
`controllerID: "ecoMAX810P-L TOUCH"`

**Controller software versions**
Module ecoNET version:  `softVer: "3.2.3819"`
Module A version: `moduleASoftVer:  "8.30.176.R1"`
Module Panel version: `modulePanelSoftVer: "8.11.42"`
Module B version: `moduleBSoftVer: "1.30.7"`
Module Lambda version: `moduleLambdaSoftVer: "3.2.7"`

**Comment by pblxptr** [comment]
EcoNet version has been already added on dev branch, so it will be available once the dev is merged. As for the rest versions I'm not sure how they should be integrated into HA, perhaps as attributes? I am not sure.

**Comment by jontofront** [comment]
I try to fnt in google documentation about econet keys, check what I found:
https://github.com/denpamusic/PyPlumIO/blob/main/pyplumio/const.py

**Comment by jontofront** [comment]
It can be add in there 
![image](https://user-images.githubusercontent.com/37038426/212302389-c87ba214-e605-4475-abdf-f2b6ebc04db4.png)

**Comment by pblxptr** [comment]
I will probably add it, actually I have already done it, it's on the dev branch together with other changes but I will probably create a separate PR for this purpose. However, I think that I will limit it only to ecoNet fw version, otherwise it would result in multiple devices being registered in device registry in Home Assistant and we don't want it.
